src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,44 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.dao.Injector;
+import mate.academy.models.Book;
+
 public class Main {
+    public static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        Book book = new Book();
```

Comment:
You could create a separate method for book creation e.g createBook() as the logic for it is repeated.